src/main/kotlin/com/sourcegraph/cody/ui/CodyToolWindowFactory.kt

```diff
@@ -275,6 +280,27 @@ class WebUIHostImpl(
               .showSettingsDialog(project, AccountConfigurable::class.java)
         }
       }
+      // TODO: Delete this intercept when Cody edits UI is abstracted so JetBrains' native UI can be invoked from the
+      // extension TypeScript side through Agent.
+      "{\"command\":\"command\",\"id\":\"cody.action.command\",\"arg\":\"edit\"}" -> {
```

Comment:
This looks a bit fragile, as all parsing jsons by hand.
Are there any downsides to parse it like I did in `handleCefQuery`?

Reply:
The serialization comes from JSON.stringify so this is not particularly fragile, and this is succinct.